AUTHORS

```diff
@@ -6,3 +6,4 @@
 Google Inc.
 Abhijeeth Padarthi <rkinabhi@gmail.com>
 Alex Li <alexv.525.li@gmail.com>
+Ibrahim Elaradi <ibrahimelaradi@gmail.com>
```

Comment:
Not sure if this is needed, just going according to the [contribution guidelines](https://github.com/google/flutter.widgets/blob/6bba637bdefc1eaa81eee49f471d30cee96c4150/CONTRIBUTING.md) here, I can remove it if it's needed since the change is trivial